sdk/template/azure-sdk-template/pom.xml

```diff
@@ -26,6 +26,13 @@
   <scm>
     <url>https://github.com/Azure/azure-sdk-for-java</url>
   </scm>
+  
+    <properties>
+    <!-- Azure Container Registry temporarily skipping code coverage until tests are added -->
+    <jacoco.skip.coverage.check>true</jacoco.skip.coverage.check>
+    <!--<jacoco.min.linecoverage>0.10</jacoco.min.linecoverage>
+    <jacoco.min.branchcoverage>0.10</jacoco.min.branchcoverage>-->
+  </properties>
```

Comment:
Since this is the template pom, we should start with the configuration in parent pom. Let's remove this as well.